simtools/model/telescope_model.py

```diff
@@ -619,6 +667,22 @@ def _loadMirrors(self):
             )
         self._mirrors = Mirrors(mirrorListFile)
 
+    def _loadReferenceData(self):
+        """Load the reference data for this telescope from the DB."""
+        self._logger.debug("Reading reference data from DB")
+        db = db_handler.DatabaseHandler()
+        self._referenceData = db.getReferenceData(self.site, self.modelVersion, onlyApplicable=True)
+
+    def _loadDerivedValues(self):
+        """Load the derived values for this telescope from the DB."""
+        self._logger.debug("Reading reference data from DB")
```

Comment:
I think this should be `Reading derived data from DB`

Reply:
True.